start_admin_sshd.sh

```diff
@@ -47,9 +35,23 @@ get_user_data_keys() {
     ( IFS=$'\n'; echo "${valid_keys[*]}" )
 }
 
+declare -r PERSISTENT_STORAGE_BASE_DIR="/.bottlerocket/host-containers/current"
+declare -r SSH_HOST_KEY_DIR="${PERSISTENT_STORAGE_BASE_DIR}/etc/ssh"
+declare -r USER_DATA="${PERSISTENT_STORAGE_BASE_DIR}/user-data"
```

Comment:
It's a bit odd to me to see this declared *below* a user of the variable - was this organized this way intentionally? What your thought on this?

Reply:
I thought it would easier to read if the functions were at the top and the main logic was below (including constants as part of the main logic)

Reply:
I'd prefer that we had these constants up top. 

There aren't user input changes to them (no flag parsing) or otherwise so they may as well be declared early on. It also loads up the mental context with the constants prior to reading their uses.

There's not too much code and assumption between the functions and constants at this point so I'll leave it up to you.

Reply:
Sure, I'll go ahead and make this change here, and in the control container script as well since my [PR](https://github.com/bottlerocket-os/bottlerocket-control-container/pull/12) is still open.